src/app_service/service/activity_center/service.nim

```diff
@@ -228,47 +203,28 @@ QtObject:
     except Exception as e:
       error "Error getting unseen activity center notifications", msg = e.msg
 
-  proc markActivityCenterNotificationRead*(
-      self: Service,
-      notificationId: string,
-      markAsReadProps: MarkAsReadNotificationProperties
-      ): string =
+  proc markActivityCenterNotificationRead*(self: Service, notificationId: string) =
     try:
-      discard backend.markActivityCenterNotificationsRead(@[notificationId])
-      self.events.emit(SIGNAL_MARK_NOTIFICATIONS_AS_READ, markAsReadProps)
-      self.events.emit(SIGNAL_ACTIVITY_CENTER_NOTIFICATIONS_COUNT_MAY_HAVE_CHANGED, Args())
+      let notificationIds = @[notificationId]
+      discard backend.markActivityCenterNotificationsRead(notificationIds)
+      self.events.emit(SIGNAL_ACTIVITY_CENTER_MARK_NOTIFICATIONS_AS_READ, ActivityCenterNotificationIdsArgs(notificationIds: notificationIds))
```

Comment:
What exactly are the cases when we still need to emit `SIGNAL_ACTIVITY_CENTER_NOTIFICATIONS_COUNT_MAY_HAVE_CHANGED`? You removed it in a few places, but it's still there sometimes.

Reply:
There were extra calls, but its still needed